### PR TITLE
Adding functionality to reload the pgbouncers from config on SIGHUP

### PR DESCRIPTION
**Pull request by jvineet**
Looks like this change is working

### CODE REVIEW

File: prometheus_pgbouncer_exporter/cli.py [hunk 1]
```diff
@@ -11,6 +11,26 @@
 from .collector import PgbouncersMetricsCollector
 
 
+def read_config_file(args):
```
Comment (dbachelder) [inline prometheus_pgbouncer_exporter/cli.py]: only args.config is used... just pass that

File: prometheus_pgbouncer_exporter/cli.py [hunk 2]
```diff
@@ -11,6 +11,26 @@
...
+    # Read config file
+    config = Config()
+    try:
+        config.read(args.config)
+        logging.getLogger().info("Config file successfully read from {file}".format(file=args.config))
+    except Exception as error:
+        logging.getLogger().fatal("Unable to read config file from {file}".format(file=args.config), extra={"exception": str(error)})
+        sys.exit(1)
```
Comment (dbachelder) [inline prometheus_pgbouncer_exporter/cli.py]: having the function do sys.exit seems wrong... I would just raise..

File: prometheus_pgbouncer_exporter/cli.py [hunk 3]
```diff
@@ -31,6 +51,10 @@ def main():
 
     # Register signal handler
     def _on_sighup(signal, frame):
+        logging.getLogger().info("Received SIGHUP - Reloading pgbouncer hosts")
+        config = Config()
```
Comment (dbachelder) [inline prometheus_pgbouncer_exporter/cli.py]: use your function?

Comment (jvineet) [reply prometheus_pgbouncer_exporter/cli.py]: lol, that's what I thought I did, but clearly skipped on.

File: prometheus_pgbouncer_exporter/cli.py (hunk 2 above)
Comment (dbachelder) [reply prometheus_pgbouncer_exporter/cli.py]: This is still an exit

Comment (jvineet) [reply prometheus_pgbouncer_exporter/cli.py]: Sorry! I just copied this block over from what they had in their main function, I should have looked at it more carefully.

**Review by dbachelder — APPROVED** [review]
lgtm

**Review by mgeggie — APPROVED** [review]
LGTm

**Pull request merged**